test/es6/proxyenumremoval.js

```diff
@@ -0,0 +1,28 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+var passed = 1;
+passed &= typeof Reflect.enumerate === 'undefined';
+
+var proxy = new Proxy({}, {
+  enumerate: function() {
+    passed = 0;
+  }
+});
+for(var key in proxy);
```

Comment:
create a test case with normal properties and see that all properties are listed.
have a test with non-enumerable property.